pkgs/by-name/el/elia/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "elia";
+  version = "1.10.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "darrenburns";
+    repo = "elia";
+    rev = "refs/tags/${version}";
+    hash = "sha256-FCdY2mS80ZQFLPlcJyT0CGP4dyo766CJUg+10MGFPeU=";
```

Comment:
```suggestion
    hash = "sha256-2CKArTo/frYLTI8qFWpkMZzpDoLDPttmMy6ZQpBDXkY=";
```

Reply:
The build does not seem to work as intended with that hash. 
When you build with that hash, the sources fetched are those of `1.9.0` not `1.10.0`.
I do not seem to be getting errors when I build the package locally using the original hash, but any CI builds do.

Reply:
Actually, after a bit of digging it seems that is an issue in the commit on the `1.10.0` tag where the version is not changed in the pyproject.toml file.